docs/platform/howto/create-service-integration.rst

```diff
@@ -0,0 +1,30 @@
+Create service integration
+===========================
+
+If you're not sure what service integration is, follow :doc:`this guide <../concepts/service-integration>` for an overview. This article will guide you on how to create service integrations between different Aiven services and move telemetry data using these integrations. To get started, you will need three services:
+
+- Aiven for Apache Kafka® - The service that produces the telemetry data
```

Comment:
```suggestion
- Aiven for Apache Kafka® - The service that produces the telemetry data. We're using a Kafka service in our example, but this might be one of the other Aiven services available.
```
I don't remember enough to know which services support integrations? So I've left this deliberately vague.